Gemfile

```diff
@@ -29,20 +29,25 @@ else
   gem 'actionpack', gem_version
 end
 
+# https://github.com/bundler/bundler/blob/89a8778c19269561926cea172acdcda241d26d23/lib/bundler/dependency.rb#L30-L54
+# https://github.com/bundler/bundler/blob/89a8778c19269561926cea172acdcda241d26d23/lib/bundler/current_ruby.rb#L118-L216
+# https://github.com/bundler/bundler/blob/1b3eb2465a64d938490110e84b5ca97a51253d14/lib/bundler/constants.rb#L2
+# require 'rbconfig'
+# !!(RbConfig::CONFIG['host_os'] =~ /cygwin|mswin|mingw|bccwin|wince|emx/) # windows
+@windows_platforms = [:mswin, :mingw, :x64_mingw]
```

Comment:
According to https://github.com/bundler/bundler/blob/89a8778c19269561926cea172acdcda241d26d23/lib/bundler/current_ruby.rb#L118-L216, we could get away with simply `:mswin`, or have I missed something?